server.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"api-booking-time/config"
+	"api-booking-time/infrastructure/datastore"
+)
+
+var (
+	db *datastore.DbRepository
+)
```

Comment:
By convention, single var declarations shouldn't be included inside a var block.

```suggestion
	var db *datastore.DbRepository
```